tests/integration/test_compute_flow.py

```diff
@@ -182,21 +248,16 @@ def run_compute_test(
         )
 
     # Start compute job
-    _, compute_start_endpoint = DataServiceProvider.build_compute_endpoint(
-        ocean_instance.config.provider_url
-    )
-    # TODO: Consider replacing with `ocean.compute.start()` after OceanCompute API is fixed.
-    job_info = DataServiceProvider.start_compute_job(
-        service_endpoint=compute_start_endpoint,
-        consumer=consumer_wallet,
-        dataset=dataset,
-        compute_environment="not implemented in provider yet",
-        algorithm=algorithm,
-        algorithm_meta=algorithm_meta,
-        algorithm_custom_data=algorithm_algocustomdata,
-        input_datasets=additional_datasets,
+    job_id = ocean_instance.compute.start(
+        ocean_instance.config.provider_url,
+        consumer_wallet,
+        dataset,
+        "not implemented in provider yet",
```

Comment:
please add a TODO here, so we can easily track this.

Reply:
Done.